content/ConnectingToDatabase/ConnectingToDatabase.facts.md

```diff
@@ -0,0 +1,60 @@
+### Facts for Connecting to Database lesson:
+
+`PHP Data Objects (PDO)` defines an interface in PHP to access certain databases
+
+`PDO` class represents a connection between a database server and PHP.
+
+`PDO_PGSQL` is a driver of `PHP Data Objects (PDO) Interface` that allows access between PHP and PostgreSQL database.
+
+`PDO_PGSQL DSN` is the PostgreSQL data source name that carries the database parameters to establish a connection. It is composed of the following DSN parameters:
+
+- `pgsql:` is the DSN prefix.
+    
+- `host` refers to the server’s hostname of the PostgreSQL database.
+
+- `port` refers to the port number of the PostgreSQL database server.
+    
+- `dbname` is the database name.
+
+- `user` refers to the username of the user.
```

Comment:
let us clearly state that this user is the user that we will use to connect to the DB. Saying just `user` is a bit confusing, is it my OS user? my email user?